diesel/src/query_dsl/mod.rs

```diff
@@ -1,18 +1,13 @@
 mod belonging_to_dsl;
-#[doc(hidden)]
```

Comment:
These should all still be `#[doc(hidden)]`. I don't want to expose the internal structure of this module, just have it be public for documentation purposes.